src/layouts/EditHomepage.jsx

```diff
@@ -43,22 +43,59 @@ import {
 import { HomepageStartEditingImage } from "assets"
 import { DEFAULT_RETRY_MSG } from "utils"
 
-import { DragDropContextProvider } from "../contexts/DragDropContext"
+import { EditableContextProvider } from "../contexts/EditableContext"
 import { useDrag, onCreate, onDelete } from "../hooks/useDrag"
 
 import { CustomiseSectionsHeader, Editable } from "./components/Editable"
 import { AddSectionButton } from "./components/Editable/AddSectionButton"
 import { HeroBody } from "./components/Homepage/HeroBody"
+import { HeroDropdownSection } from "./components/Homepage/HeroDropdownSection"
+import { HeroHighlightSection } from "./components/Homepage/HeroHighlightSection"
 import { InfobarBody } from "./components/Homepage/InfobarBody"
 import { InfopicBody } from "./components/Homepage/InfopicBody"
 import { ResourcesBody } from "./components/Homepage/ResourcesBody"
 
 /* eslint-disable react/no-array-index-key */
 
-// Constants
-// ==========
 const RADIX_PARSE_INT = 10
 
+// NOTE: `scrollIntoView` does not work when called synchronously
+// to avoid this problem, we do a `setTimeout` to wrap it.
+// This calls it after 1ms, which allows it to work.
+const scrollTo = (ref) => {
+  setTimeout(() =>
```

Comment:
where is the setTimeout's time delay here? wouldn't this execute immediately?

Reply:
Yeap! When omitting the time delay or when it's less than 1, it defaults to 1 (millisecond). 

This is the behaviour that we want - when a user expands an accordion item, they scroll to that section immediately!

Reply:
makes sense!